modules/32_power/IntC/equations.gms

```diff
@@ -68,39 +69,37 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2020 ) ..
   =l=
   sum(te2rlf(teStor,rlf),
     vm_capFac(t,regi,teStor)
-  * pm_dataren(regi,"nur",rlf,teStor)
   * vm_cap(t,regi,teStor,rlf)
   )
 ;
 
 
-*** H2 storage implementation: Storage technologies (storspv, storwind etc.) also
-*** represent H2 storage. This is implemented by scaling up capacities of 
-*** H2 turbines (h2turbVRE, seh2 -> seel) with VRE capacities which require storage (according to q32_limitCapTeStor). 
+*** H2 storage implementation:
+*** Storage technologies (storspv, storwind etc.) also represent H2 storage.
+*** This is implemented by scaling up capacities of H2 turbines (h2turbVRE, seh2 -> seel)
+*** with VRE capacities which require storage (according to q32_limitCapTeStor). 
 *** These H2 turbines (h2turbVRE) do not have capital cost. Their cost are already considered in storage technologies.
-*** H2 turbines do not need be built if sufficient gas turbines (ngt) are available to provide flexibility. 
-*' Require a certain capacity  of either hydrogen or gas turbines as peaking backup capacity. The driver is the testor capacity, which in turn is determined by v32_storloss 
+*** H2 turbines are not needed if sufficient gas turbines (ngt) are available to provide flexibility. 
+*' Require a certain capacity of either hydrogen or gas turbines as peaking backup capacity. The driver is the teStor capacity, which in turn is determined by v32_storloss 
 q32_h2turbVREcapfromTestor(t,regi)..
   vm_cap(t,regi,"h2turbVRE","1")
   + vm_cap(t,regi,"ngt","1")
   =g=
-  sum(teStor, 
-    p32_storageCap(teStor,"h2turbVREcapratio") * vm_cap(t,regi,teStor,"1") )
+  sum(teStor, p32_storageCap(teStor,"h2turbVREcapratio") * vm_cap(t,regi,teStor,"1") )
```

Comment:
Hm, I do think it is clearer if the parts "what set elements does the sum go over" and "what values are summed" are split over two lines.
This is also the way most sums are implemented, so I would disagree with changing this default. 

If you see it differently, let's maybe discuss offline?

Reply:
I've seen both, but don't mind either way!
All 3 teStor sums are now over 2 lines.